### DISCUSSION TITLE
[Feature] Some possible improvements

### DISCUSSION THREAD

**Discussion by juanpeich** [immich-app/immich#11667] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [x] Yes

### The feature

After using Immich, I would like to comment on some suggestions to implement in your fabulous application.

**Sharing photos with users:**
- Once the photos have been shared on the other user's timeline, you can see with a badge which photos belong to each member. This way you can quickly see which photos each one took, for example on a vacation trip.

- Option to have permissions to edit metadata, delete photos, etc. of the user who has shared their photos with me.

**Mobile application:**
- Viewing the photos from "Map", being able to delete them from there, without having to search for the photo again in "photos"

- Being able to rotate the photo, being able to choose to overwrite the original, so that duplicate photos are not generated.
- Synchronizing with mobile gallery: You can synchronize with the phone's gallery, so that when you delete a photo from Immich, it is also deleted from the internal gallery of the mobile phone.

**Server:**
- In the future, add the ability to put photos into the hosting where immich automatically recognizes them using samba or file explorer. Right now there is the option of Immich-Go, but you need to have some computer knowledge.

Thank you very much for your great effort and dedication in this project

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
As noted in the template, only one request per post please. Furthermore, some of these are duplicates.

**Reply by juanpeich** [reply]
Ok, sorry.

I'm going back to creating one by one?  It was to avoid creating multiple posts.
